pkg/sessions/utils/envelope/constants.go

```diff
@@ -0,0 +1,11 @@
+package envelope
+
+const (
```

Comment:
Is there a better way to expose constants like these that result in lightweight int types in the MessagePack encoded message for size? Not sure I'm in love with this organization.